sros2/xml_cache/xml.xsd

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0'?>
```

Comment:
Do we need to add some attribution to this file?  I obviously didn't create it.  It's from http://www.w3.org/2001/03/xml.xsd

Reply:
I'm not sure. At minimum, it's probably a good idea to add a reference to the source for posterity.
@ros2/team Suggestions?